mitiq/calibration/calibrator.py

```diff
@@ -37,12 +36,19 @@
     Observable,
     QuantumResult,
 )
-from mitiq.calibration.settings import Settings, Strategy
+from mitiq.calibration.settings import Settings, Strategy, DefaultStrategy
 
 
 class Calibrator:
-    """A calibration object which keeps track of, and aids in Error
-    Mitigation parameter tuning."""
+    """An object used to orchestrate experiments, and determine in Error
+    Mitigation parameter tuning.
+
+    Args:
+        executor: Noisy simulation/hardware on which to run circuits.
+        settings: A ``Settings`` object which specifies the type and amount of
+            circuits/error mitigation methods to run.
+        ideal_executor: TODO
```

Comment:
We don't currently use the `ideal_executor` for anything. Happy to keep it in if we plan on using it, but otherwise I think we should remove it so as to not confuse users.

Reply:
Good question! I think we should keep `ideal_executor` because the user may want to use an optimized backend for larger calibration "jobs". They may also want to see ideal expectation values obtained during the calibration for troubleshooting/characterizing their use case.